src/main/java/frc/robot/Robot.java

```diff
@@ -25,12 +26,14 @@
  * project.
  */
 public class Robot extends LoggedRobot {
-    private RobotContainer robotContainer;
```

Comment:
We would like to not delete RobotContainer here. EndgameSimIO should go in the RobotContainer.java class itself